tron/eventbus.py

```diff
@@ -140,6 +140,7 @@ def _has_event(self, event_id):
 
     def sync_load_log(self):
         started = time.time()
+        # we are loading the pickle object from the "current" file
```

Comment:
imo, we can probably remove this comment as its mostly restating what the code is doing - not why (or adding additional information)